### PR TITLE
[libyuv] Add libyuv

### PR DESCRIPTION
**Pull request by UnaNancyOwen**
This pull request will add [libyuv](https://chromium.googlesource.com/libyuv/libyuv).

### CODE REVIEW

File: ports/libyuv/CONTROL [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Source: libyuv
+Version: fec9121
+Build-Depends: libjpeg-turbo
```
Comment (cbezault) [inline ports/libyuv/CONTROL]: You likely don't depend on the default features, please specify exactly what features you depend on.

File: ports/libyuv/Findlibyuv.cmake.in [hunk 2]
```diff
@@ -0,0 +1,48 @@
+find_path(libyuv_INCLUDE_DIR
```
Comment (cbezault) [inline ports/libyuv/Findlibyuv.cmake.in]: Why are you providing a CMake Module? Shouldn't you be providing a cmake config file?

File: ports/libyuv/vcpkg-cmake-wrapper.cmake [hunk 3]
```diff
@@ -0,0 +1 @@
+include(${CMAKE_CURRENT_LIST_DIR}/Findlibyuv.cmake)
```
Comment (cbezault) [inline ports/libyuv/vcpkg-cmake-wrapper.cmake]: If Kitware doesn't provide a built in module there is no reason to do this and you should make a config file.

**Review by cbezault — CHANGES_REQUESTED** [review]

File: ports/libyuv/CONTROL (hunk 1 above)
Comment (UnaNancyOwen) [reply ports/libyuv/CONTROL]: yuvconvert requires libjpeg (or libjpeg-turbo).

File: ports/libyuv/Findlibyuv.cmake.in (hunk 2 above)
Comment (UnaNancyOwen) [reply ports/libyuv/Findlibyuv.cmake.in]: I see. I will change this port to provide it.

File: ports/libyuv/vcpkg-cmake-wrapper.cmake (hunk 3 above)
Comment (UnaNancyOwen) [reply ports/libyuv/vcpkg-cmake-wrapper.cmake]: I was changed this port to provide it.

**Comment by UnaNancyOwen** [comment]
@cbezault Please re-review! Thanks,

File: ports/libyuv/CONTROL (hunk 1 above)
Comment (cbezault) [reply ports/libyuv/CONTROL]: What I was suggesting was that you write something like this: `libjpeg-turbo[core, feature1, feature2, ...]`, it's fine for now but we're probably going to want to change it in the future.

**Review by cbezault — APPROVED** [review]

**Pull request merged**